# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaCodecVideoCodec.cpp
# Commit: 9d8f6e354d8c
# Full Hash: 9d8f6e354d8c332bab7e433252077ec0e1fa8f04
# Author: John Lin <jolin@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - remove MediaCodecVP8Codec. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115531
# ==============================================================================

diff -r 7675ead797c8 -r 9d8f6e354d8c dom/media/webrtc/libwebrtcglue/MediaCodecVideoCodec.cpp
--- a/dom/media/webrtc/libwebrtcglue/MediaCodecVideoCodec.cpp	Fri Oct 08 15:57:53 2021 +0200
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,41 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "common/browser_logging/CSFLog.h"
-#include "nspr.h"
-#include "mozilla/StaticPrefs_media.h"
-
-#include "WebrtcMediaCodecVP8VideoCodec.h"
-#include "MediaCodecVideoCodec.h"
-
-namespace mozilla {
-
-static const char* mcvcLogTag = "MediaCodecVideoCodec";
-#ifdef LOGTAG
-#  undef LOGTAG
-#endif
-#define LOGTAG mcvcLogTag
-
-WebrtcVideoEncoder* MediaCodecVideoCodec::CreateEncoder(CodecType aCodecType) {
-  CSFLogDebug(LOGTAG, "%s ", __FUNCTION__);
-  if (aCodecType == CODEC_VP8) {
-    if (StaticPrefs::
-            media_navigator_hardware_vp8_encode_acceleration_remote_enabled()) {
-      return new WebrtcMediaCodecVP8VideoRemoteEncoder();
-    } else {
-      return new WebrtcMediaCodecVP8VideoEncoder();
-    }
-  }
-  return nullptr;
-}
-
-WebrtcVideoDecoder* MediaCodecVideoCodec::CreateDecoder(CodecType aCodecType) {
-  CSFLogDebug(LOGTAG, "%s ", __FUNCTION__);
-  if (aCodecType == CODEC_VP8) {
-    return new WebrtcMediaCodecVP8VideoDecoder();
-  }
-  return nullptr;
-}
-
-}  // namespace mozilla